src/worker.cc

```diff
@@ -399,6 +399,14 @@ void parse_locations(const rapidjson::Document& doc,
         if (preferred_side && PreferredSide_Parse(*preferred_side, &side)) {
           location->set_preferred_side(side);
         }
+
+        lat = rapidjson::get_optional<float>(r_loc, "/display_lat");
+        lon = rapidjson::get_optional<float>(r_loc, "/display_lon");
+        if (lat && lon && *lat >= -90.0f && *lat <= 90.0f) {
+          lon = midgard::circular_range_clamp<float>(*lon, -180, 180);
+          location->mutable_display_ll()->set_lat(*lat);
+          location->mutable_display_ll()->set_lng(*lon);
+        }
```

Comment:
this is where we parse the request from json into our internal pbf request object